modules/data/about_prefs.components.json

```diff
@@ -443,5 +443,42 @@
         "selectorData": "richlistitem[type='application/pdf'] menulist.actionsMenu",
         "strategy": "css",
         "groups": []
+    }, 
+
+    "use-primary-password": {
+        "selectorData": "useMasterPassword",
+        "strategy": "id",
+        "groups": []
+    },
+
+    "current-pw": {
```

Comment:
Lets try and not shorten terms unless we need to. I am assuming `pw` means password? We can spell `password` out for these instead.

Reply:
Changed as well